src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -329,9 +329,19 @@
 
   :handle-ok
   (fn [context]
-    (->
-      (ca/get-certificate-statuses settings)
-      (as-json-or-pson context))))
+    (let [query-params (:params request)
```

Comment:
`:params` doesn't include the query params when testing.
I was originally using a `:query-params` key, but that key doesn't exist at all in the `request` from testing.